hardware/raspberrypi/revision-codes/README.md

```diff
@@ -0,0 +1,94 @@
+# Raspberry Pi revision codes
+
+Each distinct Raspberry Pi model revision has a unique revision code. You can look up a Raspberry Pi's revision code by running:
+
+```bash
+cat /proc/cpuinfo
+```
+
+The last three lines show the hardware type, the revision code and the Pi's unique serial number, for example:
+
+```
+Hardware    : BCM2835
+Revision    : a02082
+Serial      : 00000000765fc593
+```
+
```

Comment:
It might worth adding a quick note to say that, as of the switch to the 4.9 kernel, all Pis will report `BCM2835` - even those with BCM2836 and BCM2837 processors - and that this is as a result of using more upstream code. Any applications still using the `Hardware` string to detect Pi2 and Pi3 have to transition to using the Revision code.